src/tasks/FishingTask.py

```
@@ -50,8 +50,7 @@ def __init__(self, *args, **kwargs):
         )
         self.config_description.update(
             {
-                self.CONF_CONTROL_MODE: f"{self.MODE_HOLD}：平滑流畅, 易过冲\n"
-                f"{self.MODE_TAP}: 安全较慢, 防过冲",
+                self.CONF_CONTROL_MODE: self.tr("{}：平滑流畅, 易过冲\n{}: 安全较慢, 防过冲").format(self.MODE_HOLD, self.MODE_TAP),
```

Ruff (0.15.13)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)